generated/json_data/skippable_cutscenes_pal.jsonc

```diff
@@ -99,6 +99,11 @@
                         589935,
                         589932
                     ]
+                },
+                "Hall of the Elders": {
+                    "deleteIds": [
```

Comment:
This isn't mentioned in your changelog

Reply:
uh I think I did something wrong, I did not want to include this file

Reply:
This PR has two commits in it. It looks like you accidentally based your changes off this one:
![image](https://github.com/toasterparty/randomprime/assets/14116556/09079480-4ee7-49fc-915f-bd90c4a5d8c0)

Please revert these changes or justify why they should stay

Reply:
this is a test comment to test the new discord integration settings

Reply:
Well these changes were supposed to go in the previous PR, you're telling me they didn't get in?